PayPalUI/src/main/java/com/paypal/android/ui/paymentbutton/PayLaterButton.kt

```diff
@@ -0,0 +1,56 @@
+package com.paypal.android.ui.paymentbutton
+
+import android.content.Context
+import android.os.Build
+import android.util.AttributeSet
+import androidx.annotation.RequiresApi
+import com.paypal.android.ui.R
+
+
+/**
+ * PayLaterButton provides a PayPal PayLater button with the ability to modify the [color], [label], [shape],
+ * and [size].
+ *
+ * Setting up PayLaterButton within an XML layout:
+ * ```
+ * <PayPalLater
+ *      android:id="@+id/payLaterButton"
+ *      android:layout_width="match_parent"
+ *      android:layout_height="wrap_content" />
+ * ```
+ *
+ * Optionally you can provide the following attributes: `paypal_color`, `paypal_label`,
+ * `payment_button_shape`, and `payment_button_size`.
+ */
+@RequiresApi(Build.VERSION_CODES.M)
```

Comment:
Curious what in here requires API 23? Should we be considering raising the min SDK of the entire SDK to 23? It feels odd to have a button that isn't available to all SDK users.

Reply:
i think we are taking this from NXO, but i tested it and we dont seem to need it. I've removed it